.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: android
```

Comment:
this project is not related to android

Reply:
Whoa, I'm so sorry, the project file structure and files looked just android. I can try to resolve the issue to make it work.